scripts/check-examples.ts

```diff
@@ -25,6 +25,7 @@ for (const dir of readdirSync(examplesDir)) {
     dir,
   )
   runProcess([yarn, 'typecheck'], dir)
+  runProcess([yarn, 'test'], dir)
```

Comment:
We go back to the question "How much are the examples actually end-to-end tests?"

Having complex tested examples certainly hinders their merit as easy learning reference, but it also ensures that they keep working and the presented capabilities are still supported.

Reply:
Hmm i think that right now there are still very simple (ie good learning material). What's wrong in treating them as sort of e2e tests too?

Reply:
Agreed. I think about examples as 1 and 2 but not 3 (but it usually requires minimal tweaks to extract it from the repo).